src/scripts/migrate.py

```diff
@@ -121,12 +124,14 @@ def slurp(
             next_mondo_id, mondo_term_ids = _get_next_available_mondo_id(next_mondo_id, max_id, mondo_term_ids)
             mondo_id = 'MONDO:' + str(next_mondo_id).zfill(7)  # leading 0-padding
         mondo_label = t.label.lower() if t.label else ''
+        qualified_parents = [p for p in t.direct_owned_parent_curies
+                             if p in match_types and match_types[p] in ['skos:exactMatch', 'skos:narrowMatch']]
+        qualified_mondo_paents = [mondo_id_map[p] for p in qualified_parents if p in mondo_id_map]
         terms_to_slurp.append({
             'mondo_id': mondo_id, 'mondo_label': mondo_label, 'xref': t.curie, 'xref_source': 'MONDO:equivalentTo',
             'original_label': t.label if t.label else '', 'definition': t.definition if t.definition else '',
             # if not in match_types, this should mean term is excluded or obsolete
-            'parents': '|'.join([p for p in t.direct_owned_parent_curies if p in match_types
-                                 and match_types[p] in ['skos:exactMatch', 'skos:narrowMatch']])})
+            'parents': '|'.join(qualified_mondo_paents)})
```

Comment:
[Bugfix](https://github.com/monarch-initiative/mondo-ingest/pull/319#discussion_r1208114788): Robot template 'parents' column IDs were not Mondo IDs